### PR TITLE
Initial instructions

### PR DESCRIPTION
**Pull request by lindsayplatt**
Why hello! I've been working away on the instructions and want to merge in the work so far to avoid this being too big of a PR. So, this PR creates an initial set of instructions for following along with the validation and labeling workflows (I combined the two since they are so similar). I am currently having an issue with getting images to render (see below) and will leave as draft until that rendering issue is resolved.

As for issues that this touches:
- I believe this completes #10 (see the tab called `Initial setup`). Though, I did delete [a step that the GROD workflow noted about creating a Git repo](https://github.com/GlobalHydrologyLab/GROD/blob/master/docs/index.Rmd#L75) because I don't remember that option ever popping up for me.
- It is also the beginning of #14 and #17 (see the tabs `How to label`). I think I need to see more of the validation and labeling script process (maybe we should do a quick call this week?) before I can make more progress on those instructions. There will be more to add about the difference between them, I'm sure.
- It leaves a placeholder section for #13 called `What are we labeling?`

image

For rendering, I have my Global Options for RMarkdown set to render relative to the `Project`. As such, I've edited the image filepaths to include `docs/`. If I don't do this, I get an error about not finding the file when I try to knit. When I added `docs/` I get passed the error but get a warning that says `[WARNING] Could not fetch resource` and I can't see the images. Any ideas?

### CODE REVIEW

File: docs/index.Rmd [hunk 1]
```diff
@@ -10,21 +10,142 @@ output:
...
+knitr::include_graphics("docs/www/images/categories.png")
+```
+
+2. When you see a new feature in the image that you would like to label, you will need to click on the top left geometry bar and select the point geometry icon.
+
+```{r, echo=FALSE, out.width = '50%'}
+knitr::include_graphics("docs/www/images/addmarker.png")
+```
+
+3. Change the category to the type of feature that you are labeling by hovering over `openWater` (it just chooses the first category by default) and clicking the name of the category you want.
```
Comment (steeleb) [inline docs/index.Rmd:114]: maybe pixel here too?

File: docs/index.Rmd [hunk 2]
```diff
@@ -10,21 +10,142 @@ output:
...
+
+### Adding labels
+
+1. Check that you have the appropriate geometry categories by hovering over `Geometry Imports` in the map area of GEE. If you don't see a list of sediment and bloom types, then you should revisit Part 1 and follow those instructions carefully.
+
+```{r, echo=FALSE, out.width = '80%'}
+knitr::include_graphics("docs/www/images/categories.png")
+```
+
+2. When you see a new feature in the image that you would like to label, you will need to click on the top left geometry bar and select the point geometry icon.
```
[the remark this answers is not in the document]
Comment (steeleb) [reply docs/index.Rmd:108]: And maybe we add context: 'For each tile that you are working on, you will have to zoom in until you can discern pixels. This is especially important near shore.' Or something like that?

**Comment by steeleb** [comment]
Rockstar.

> I believe this completes https://github.com/rossyndicate/Superior-Plume-Bloom/issues/10 (see the tab called Initial setup). Though, I did delete [a step that the GROD workflow noted about creating a Git repo](https://github.com/GlobalHydrologyLab/GROD/blob/master/docs/index.Rmd#L75) because I don't remember that option ever popping up for me.

Agreed, this closes #10. And I think that the GEE UI automatically does git versioning now, so I agree, it's unnecessary.

> I think I need to see more of the validation and labeling script process (maybe we should do a quick call this week?) before I can make more progress on those instructions. There will be more to add about the difference between them, I'm sure.

Superb! I'll follow up via email for a call. And yes, I'm working on these today and can get us (royal us, really: you) in a better spot since everything on my end took 1000% longer than I anticipated because of the differences between GROD and eePlumB. I should be making progress on #12, #15, #19, #20 today.

> For rendering, I have my Global Options for RMarkdown set to render relative to the Project. As such, I've edited the image filepaths to include docs/. If I don't do this, I get an error about not finding the file when I try to knit. When I added docs/ I get passed the error but get a warning that says [WARNING] Could not fetch resource and I can't see the images. Any ideas?

Okay, I really have no idea why the `knitr::include_graphics()` isn't working I tried a few things, too, with no great solution. This is the workaround I used that seems to render correctly:

image

In source code it's just:
``

This knits to:
image

File: docs/index.Rmd [hunk 3]
```diff
@@ -10,21 +10,142 @@ output:
...
+
+Now that you have the script running, you are ready to add labels. Head to the next section, `How to label: Part 2`!
+
+## How to label: Part 2
+
+The purpose of this step is to teach you how to use our Google Earth Engine framework to classify images. Hopefully, this section is an easy reference if you need a refresher in the future. Note that this step assumes you have already done `How to label: Part 1` for the current set of images (either validation or for the specific mission-date you have been assigned).
+
+### Adding labels
+
+1. Check that you have the appropriate geometry categories by hovering over `Geometry Imports` in the map area of GEE. If you don't see a list of sediment and bloom types, then you should revisit Part 1 and follow those instructions carefully.
```
[the remark this answers is not in the document]
Comment (lindsayplatt) [reply docs/index.Rmd:102]: That list isn't top right for me, it's top left. Interesting!

[the remark this answers is not in the document]
Comment (steeleb) [reply docs/index.Rmd]: I'm a derp. It's 100% top left for me too.

**Pull request merged**